api/proto/teleport/notifications/v1/notifications.proto

```diff
@@ -80,6 +80,9 @@ message GlobalNotificationSpec {
     ByRoles by_roles = 2;
     // all represents whether to target all users, regardless of roles or permissions.
     bool all = 3;
+    // by_users represents a list of usernames of the users targeted by this notification.
+    // If only one user is being targeted, please create a user-specific notification instead.
+    ByUsers by_users = 7;
```

Comment:
how did users get notifications like this before this field?

Reply:
we just didn't support sending notifications to multiple users via a list of usernames before. We have user-specific notifications, but they only support targeting 1 user. This field allows us to create a global notification that can target multiple users by usernames.